derp/derp_server.go

```diff
@@ -453,7 +474,11 @@ func (s *Server) initMetacert() {
 		// Windows requires NotAfter and NotBefore set:
 		NotAfter:  time.Now().Add(30 * 24 * time.Hour),
 		NotBefore: time.Now().Add(-30 * 24 * time.Hour),
+		// macOS requires BasicConstraints when subject == issuer:
```

Comment:
Colon here plus parens on next line is weird.

Reply:
Done.